### PR TITLE
Makes starting a vote respect stealthmin privacy

### PR DESCRIPTION
**Pull request by silicons**
REEEEEEE!!!

### CODE REVIEW

**Comment by Shapsy** [comment]
This is clearly not what stealthmin is for, if you spam votes you should be held accountable for them

**Comment by PKPenguin321** [comment]
^ stop misusing stealthmin to afk and meme

**Comment by RussellTalbott** [comment]
i love getting a PM from Administrator because I know I pissed in Kev's mouth somehow

**Comment by PKPenguin321** [comment]
>when kevin uses stealthmin so commonly that you automatically know it's him when you get a stealth PM

jesus kevin what the fuck

File: code/controllers/subsystem/voting.dm [hunk 1]
```diff
@@ -163,6 +163,10 @@ var/datum/subsystem/vote/SSvote
 			var/ckey = ckey(initiator_key)
 			if((admin_datums[ckey]) || (ckey in deadmins))
 				admin = TRUE
+				if(admin_datums[ckey])
+					var/datum/admins/A = admin_datums[ckey]
+					if(A.fakekey)
+						initiator_key = "Administrator"
```
Comment (MrStonedOne) [inline code/controllers/subsystem/voting.dm]: no, use the fake key here.

Comment (MrStonedOne) [inline code/controllers/subsystem/voting.dm]: unless it would require admin status to even do, then use Administrator

**Pull request closed without merging**